tests/unit/dataactbroker/test_dashboard_handler.py

```diff
@@ -193,7 +194,11 @@ def setup_submissions(sess, admin=False):
     rule_ab2 = RuleSql(rule_sql='', rule_label='B1', rule_error_message='second cross rule', query_name='',
                        file_id=FILE_TYPE_DICT_LETTER_ID['B'], target_file_id=FILE_TYPE_DICT_LETTER_ID['A'],
                        rule_severity_id=RULE_SEVERITY_DICT['warning'], rule_cross_file_flag=True, category='existence')
-    db_objects.extend([rule_a1, rule_a2, rule_ab1, rule_ab2])
+    # Adding a rule with the same rule label as another but different file
+    rule_cd1 = RuleSql(rule_sql='', rule_label='B1', rule_error_message='extra cross rule', query_name='',
```

Comment:
None of our existing "same label" rules are for the same job. One is for file B and the other is for file C. Should we be testing with that in mind?